lib/matplotlib/axes/_base.py

```diff
@@ -391,8 +371,6 @@ def _grab_next_args(self, *args, **kwargs):
 
 
 class _AxesBase(martist.Artist):
-    """
```

Comment:
Was this maybe here to prevent docstring inheritance? OTOH, doesn't make much sense since the class is private and thus neither shows up in the HTML docs nor should a user encounter it in an interactive session. So probably fine.